### PR TITLE
NMS-16490 : Installation instructions are broken and show an unresolved variable

### PR DESCRIPTION
**Pull request by MusaidAli**
### External References

* Jira (Issue Tracker): https://opennms.atlassian.net/browse/NMS-16490

### CODE REVIEW

**Comment by MusaidAli** [comment]
Screenshot from 2024-10-18 14-46-10

File: docs/modules/deployment/pages/core/centos-rhel7/install-core.adoc [hunk 1]
```diff
@@ -8,9 +8,9 @@ endif::[]
 
 .Install {page-component-title} with all built-in dependencies
 [source, console]
```
Comment (cgorantla) [inline docs/modules/deployment/pages/core/centos-rhel7/install-core.adoc:10]: remove [source,console]

and use ** 

`==` is used for headings not for commands.

Comment (MusaidAli) [reply docs/modules/deployment/pages/core/centos-rhel7/install-core.adoc]: Screenshot from 2024-10-21 14-27-02
yes it resolved.

**Review by cgorantla — COMMENTED** [review]
@indigo423  Can you review this, may be there are better alternatives in antora for replacing placeholders in code blocks

**Comment by indigo423** [comment]
We need to go back to fixing a regression in foundation-2024, see #7524.

**Pull request closed without merging**